### ISSUE TITLE
Possible new topic area: sustainability

### ISSUE THREAD

**Issue by bernhold** [betterscientificsoftware/bssw.io#591]
In the BSSw.io topic map, we don't really have anything specifically for software sustainability.  The closest is probably software process improvement or maybe just software engineering.  Some people, like Dan Katz have a very specific focus on that topic.

So should we add "Sustainability" to the topic map?

**Comment by bernhold** [comment]
To be concrete, I propose that we add Better Planning > Software Sustainability.  That seems to me to be the most logical category.  Please comment, then let's implement!

**Comment by markcmiller86** [comment]
So, I don't think you get notifications when someone adds a reaction to your comment but I just added a 👍🏻 to it 😉

**Comment by bartlettroscoe** [comment]
I agree, I think it would be good to add a "software sustainability" topic.

**Comment by bernhold** [comment]
Agreed at 2022-06-30 EB meeting.  David will implement
